newrelic-weaver/src/main/java/com/newrelic/weave/MethodCallInlinerAdapter.java

```diff
@@ -234,6 +239,57 @@ protected int newLocalMapping(final Type type) {
         }
     }
 
+    /**
+     * Flags method nodes requiring additional return insn processing, which for now is only invokeSuspend.
+     * For future reference, if other code surfaces a bytecode verification failure (such as an ArrayIndexOutOfBoundsException),
+     * modify this guard to process additional methods beyond invokeSuspend.
+     */
+    private boolean shouldClearReturnStacks(String name, String desc) {
+        if (clearReturnStacksDisabled()) {
+            return false;
+        }
+        final String invokeSuspendName = "invokeSuspend";
+        final String invokeSuspendDesc = "(Ljava/lang/Object;)Ljava/lang/Object;";
+        return invokeSuspendName.equals(name) && invokeSuspendDesc.equals(desc);
+    }
+
+    /**
+     * Feature flag to disable return stack processing.
+     */
+    private boolean clearReturnStacksDisabled () {
+        String featFlag = System.getProperty("newrelic.config.class_transformer.clear_return_stacks");
```

Comment:
```suggestion
        String featFlag = System.getProperty("newrelic.config.class_transformer.clear_return_stacks", false);
```

I believe you can call `getProperty` with a default value of `false` which will be returned if the system property doesn't exist, rather than having it potentially return `null`. 

I also think that `getProperty` can potentially throw an exception so you should make sure to handle that.

Reply:
Refactored to have getProperty return a default value of `true` (the feature flag is an off switch with a value of `false`). And hopefully the flag checker is slightly more readable now. 

Leaving the unchecked exceptions for getProperty unhandled, to follow the pattern in the rest of the agent.